src/components/pages/contact/Contact.jsx

```diff
@@ -0,0 +1,48 @@
+import { useState, useRef } from 'react';
+import { StyledButton, StyledContainer, StyledH2, ContactBackground } from './Contact.styled';
+import { InputField } from './InputField';
+import { TextareaField } from './TextareaField';
+import emailjs from '@emailjs/browser';
+
+export const Contact = () => {
+    const form = useRef();
+
+    const [name, setName] = useState('');
```

Comment:
Tutaj zamiast 3 state, można złączyć to w 1 i manipulować odpowiednimi key w obiekcie

Reply:
Done